src/Tribe/Views/V2/Messages.php

```diff
@@ -94,6 +94,10 @@ public static function for_key( $key, ...$values ) {
 				'There were no results found.',
 				'the-events-calendar'
 			),
+			'no_upcoming_events'                 => __(
```

Comment:
:warning: Array double arrow not aligned correctly; expected 15 space(s) between "'no_upcoming_events'" and double arrow, but found 17.

<sub>hash: 66333984b2054f8ba0a2db0725b406b9</sub>